posthog/models/action/action.py

```diff
@@ -45,6 +46,18 @@ def get_analytics_metadata(self):
             "deleted": self.deleted,
         }
 
+    def get_step_events(self) -> List[str]:
+        events = []
+        for action_step in self.steps.all():
```

Comment:
Had a look at how insights do it, as it turns out, the event is populated correctly always, so we can just do

```
action = entity.get_action()
                for action_step in action.steps.all():
                    events.add(action_step.event)
```

Reply:
So cheeky